src/app/navigation/bottom-tab.tsx

```diff
@@ -1,15 +1,56 @@
 import React from 'react';
 import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 import { ExerciseScreen, HomeScreen, RoutinesScreen } from '@screens/index';
+import {
+  DumbbellIcon,
+  HomeIcon,
+  RoutineIcon,
+} from '@src/assets/svg/navigation-icons';
 
-const Tab = createBottomTabNavigator();
+export type BottomTabParamList = {
+  RoutinesScreen: undefined;
+  HomeScreen: undefined;
+  ExerciseScreen: undefined;
+};
```

Comment:
WDYT if we move all of these types into a directory called, types or interfaces inside a file called navigation. So with that approach, we'll have all the navigation types in one place.